### ISSUE TITLE
README describes yarn dev configuration as based on standalone instead of demo

### ISSUE THREAD

**Issue by amb26** [IMERSS/inat-components#9]
The "local dev env" README section at https://github.com/IMERSS/inat-components/blob/main/README.md#local-dev-env suggests that the dev configuration is based on `apps/standalone` whereas it seems it is really based on `apps/demo`?

**Comment by benkeen** [comment]
Yeah that was wrong, sorry - probably from an earlier iteration of the script.  

Just going to shuffle the code around a bit and re-document it all when I'm done. Will include this then.
